### PR TITLE
chore: run as non-root user

### PR DESCRIPTION
**Pull request by ptab**
#security 🔐

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -13,33 +14,34 @@ RUN set -eux ; \
...
 # Keeps Python from buffering stdout and stderr to avoid situations where
 # the application crashes without emitting any logs due to buffering.
 ENV PYTHONUNBUFFERED=1
 
+## Switch to the directory of the caller repo (must be mounted while running)
+WORKDIR /repo
+
 # Run the application.
-ENTRYPOINT ["python", "/app/mpyl/__main__.py"]
+ENTRYPOINT ["pipenv", "run", "python", "/app/mpyl/src/mpyl/__main__.py"]
```
Comment (Jorg88) [inline Dockerfile:46]: are these changes related to the non root user?

Comment (ptab) [reply Dockerfile:46]: Yes. Before we would [install `pipenv` dependencies to the entire system](https://github.com/Vandebron/gh-mpyl/pull/112/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557L27), now we install it only for the non-root user. This requires us to trigger the application through pipenv (like we do locally).
